File: src/main/java/cloud/api/RebalanceResponseMap.java
package cloud.api;

import cloud.common.NodeConfiguration;
import se.sics.kompics.Event;

/**
 * 
 * @author Amir Moulavi
 * @date 2011-04-21
 *
 */

public class RebalanceResponseMap extends Event {

	private NodeConfiguration nodeConfiguration;

	public RebalanceResponseMap(NodeConfiguration nodeConfiguration) {
		this.nodeConfiguration = nodeConfiguration;
	}
	
	public NodeConfiguration getNodeConfiguration() {
		return nodeConfiguration;
	}

}

File: src/main/java/cloud/api/RebalanceResponseMap.scala
package cloud.api

import cloud.common.NodeConfiguration
import se.sics.kompics.Event
import reflect.BeanProperty

/**
 * @author Amir Moulavi
 * @date 2011-07-16
 *
 */
class RebalanceResponseMap(@BeanProperty var nodeConfiguration:NodeConfiguration) extends Event